src/basho_bench_worker.erl

```diff
@@ -56,20 +60,20 @@
 %% API
 %% ====================================================================
 
-start_link(SupChild, Id) ->
-    case basho_bench_config:get(distribute_work, false) of 
```

Comment:
Not sure which style change you are referring to - this was a functional change to accept a third parameter, WorkerConf